packages/api/src/schema/api-schema.yaml

```diff
@@ -0,0 +1,3102 @@
+          type: string
+          example: de7818e7-610a-4057-8f6f-b785dc1e6f88
+        kind:
+          type: string
+          example: stream
+        name:
+          type: string
+          example: test_session
+        userId:
+          type: string
+          example: 66E2161C-7670-4D05-B71D-DA2D6979556F
+        lastSeen:
+          type: number
+          example: 1587667174725
+        sourceSegments:
+          type: number
+          example: 1
+        transcodedSegments:
+          type: number
+          example: 2
+        sourceSegmentsDuration:
+          type: number
+          example: 1
+          description: "Duration of all the source segments, sec"
+        transcodedSegmentsDuration:
+          type: number
+          example: 2
+          description: "Duration of all the transcoded segments, sec"
+        sourceBytes:
+          type: number
+          example: 1
+        transcodedBytes:
+          type: number
+          example: 2
+        ingestRate:
+          type: number
+          example: 1
+          description: Rate at which sourceBytes increases (bytes/second)
+        outgoingRate:
+          type: number
+          example: 2
+          description: Rate at which transcodedBytes increases (bytes/second)
+        broadcasterHost:
+          type: string
+          description: Hostname of the broadcaster that transcodes that stream
+        deleted:
+          type: boolean
+          description: Set to true when stream deleted
+        createdAt:
+          type: number
+          description:
+            Timestamp (in milliseconds) at which stream object was created
+          example: 1587667174725
+        parentId:
+          type: string
+          example: de7818e7-610a-4057-8f6f-b785dc1e6f88
+          description: Points to parent stream object
+        record:
+          description: >-
+            Should this stream be recorded? Uses default settings. For more
+            customization, create and configure an object store.
+          type: boolean
+          example: false
+        recordingStatus:
+          type: string
+          description: Status of the recording process of this stream session.
+          enum:
+            - waiting
+            - ready
+            - none
+        recordingUrl:
+          type: string
+          description: URL for accessing the recording of this stream session.
+        mp4Url:
+          type: string
+          description: URL for the stream session recording packaged in an mp4.
+        recordObjectStoreId:
+          type: string
+          example: D8321C3E-B29C-45EB-A1BB-A623D8BE0F65
+          description:
+            ID of object store where to which this stream was recorded
+        playbackId:
+          type: string
+          example: eaw4nk06ts2d0mzb
+          description: Used to form playback URL
+        profiles:
+          type: array
+          items:
+            $ref: "#/components/schemas/ffmpeg-profile"
+    error:
```

Comment:
Ohhh interesting! We should use this schema for the error responses in the API declarations below. Will save a lot of repetitive declarations.